### ISSUE TITLE
Throttle CLB calls in convergence gathering

### ISSUE THREAD

**Issue by manishtomar** [rackerlabs/otter#1729]
When performing #1709, 12 groups with 25 servers each with 1 CLB were converging at same time. This resulted in CLB rate limit errors since all of them were gathering all CLBs along with their nodes at same time. We should probably try to throttle the CLB requests on per tenant basis similar to the way we throttle Nova requests on a global basis.

**Comment by manishtomar** [comment]
@radix I believe you had some nicer ideas on throttling in effect.

**Comment by radix** [comment]
not really nice. here's an idea


```python
my_throttler = Throttler(interval) # or perhaps `Throttler(concurrency, interval)`
```

this is an impure invocation -- it basically creates a LoopingCall. Then, elsewhere...

```python
eff = Effect(Throttle(my_throttler, some_eff))
```

would return an Effect that has the same result as `some_eff`, but which causes the performance of `some_eff` to be throttled and serialized within the scope of `throttler`.

**Comment by manishtomar** [comment]
Sounds good.

**Comment by radix** [comment]
fixed by #1766.
